pkg/service/delete.go

```diff
@@ -37,6 +39,25 @@ func (c *DeleteCommand) Exec(in io.Reader, out io.Writer) error {
 	}
 	c.Input.ID = serviceID
 
+	if c.force {
```

Comment:
12. I discovered no one has actually tried deleting a service using the CLI as they would discover that it would fail with a message to say the activated service can't be deleted. So I've copied the technique we use in the Terraform provider which is to accept a flag to force deletion. All this does is first _disable_ the service before attempting to delete it.

Reply:
Nit: this belongs in its own PR.